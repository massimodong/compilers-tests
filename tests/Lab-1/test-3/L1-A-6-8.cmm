float BTAo [ 1585707656 ] [ 2130141265 ] ul3mClMiGvChaB 

;
float [ 261595952 ] [ 1058582620 ] 
;
struct {
}QTKKy8_D4yqcs , 

;
^] 