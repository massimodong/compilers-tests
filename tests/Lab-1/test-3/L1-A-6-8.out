Error Type B at Line 1
Error Type B at Line 4
Error Type B at Line 9
Error Type A at Line 10
Error Type B at Line 10
